src/Commands/TankDrive.cpp

```diff
@@ -47,15 +45,27 @@ void TankDrive::Interrupted() {
 
 }
 
+void TankDrive::SetAutomaticShifting(bool automaticShifting) {
+ mAutomaticShifting = automaticShifting;
+}
+
+bool TankDrive::IsAutoShifterEnabled(){
+  return mAutomaticShifting;
+}
+
 void TankDrive::AutoShifting() {
   std::shared_ptr<Chassis> chassis = Chassis::getInstance();
+
   if (chassis->IsShifterHigh()) {
+
     if (chassis->GetLeftEncRPM() <= kShiftDownVelocity
-      || chassis->GetRightEncRPM() <= kShiftDownVelocity) {
-    chassis->SetShifter(Chassis::kShifterLow);
+        || chassis->GetRightEncRPM() <= kShiftDownVelocity) {
+      chassis->SetShifter(Chassis::kShifterLow);
     }
   }
+
```

Comment:
This whitespace is confusing. Separating an `else` from the `if` makes it difficult to reason about the code.